### DISCUSSION TITLE
doris-2.02 Tablet VERSION_ERROR

### DISCUSSION THREAD

**Discussion by Semon120** [apache/doris#30104] [A - General / Q&A]
doris 2.02 里Tablet的所有副本的Status都是VERSION_ERROR，我看可能是某一次导入数据后版本有问题。tablet的LastFailedVersion是62874，LastSuccessVersion是62873。我看到最新Doris文档里有admin set replica version的语句将LastFailedVersion设置为-1，以可以正常查询数据。但doris 2.02版本好像不支持这个SQL语句，我可以用其他方式来设置这个LastFailedVersion吗？

**Comment by LemonLiTree** [comment]
你建表几副本？
